docs/development.rst

```diff
@@ -142,7 +142,7 @@ Create an environment and install the whole mss package dependencies then remove
 
   $ conda create -n mssdev mamba
   $ conda activate mssdev
-  $ mamba install mss=$mss_version --only-deps
```

Comment:
this will be replaced on build on RTFD to the current number, we should keep this, because that worked alwasys in the past.  adding python is correct ;) but keep the env var

Reply:
triggerd the build, that's still manually, the package must be first in the channel to get it installed.

Reply:
Thank you. I've fixed it. : )